### PR TITLE
Improve description of the Jira tickets

### PR DESCRIPTION
**Pull request by seilagamo**
Improvements included in this PR:
- Improve the description format, such as using bullet points or preformatted code for the detail field.
- Skip fields when they are empty.
- Remove the current exposure and status fields.
- Place the Vulcan finding URL first in the ticket description.
- Criticality is missing. Please include a field to indicate the criticality of the finding.

### CODE REVIEW

File: src/report/createTicketForm/createTicketForm.vue [hunk 1]
```diff
@@ -118,20 +119,26 @@ export default class FindingTicketCreationForm extends Vue {
 
   private buildCreateTicketDescription(finding: Finding): string {
 
-    let description: string = "";
+    let description: string = `* Vulcan URL: ${window.location.href}\n`;
```
Comment (jesusfcr) [inline src/report/createTicketForm/createTicketForm.vue:122]: I would hide the link with something like this  `[Finding|${window.location.href}]` or similar.

File: src/report/createTicketForm/createTicketForm.vue [hunk 2]
```diff
@@ -118,20 +119,26 @@ export default class FindingTicketCreationForm extends Vue {
...
 
     if (finding.issue) {
-      description += `Description: ${finding.issue.description ?? ""}\n`;
-      description += `CWE: ${finding.issue.cweId ?? ""}\n`;
+      description += `* Description: ${finding.issue.description ?? ""}\n`;
```
Comment (jesusfcr) [inline src/report/createTicketForm/createTicketForm.vue:125]: finding.issue.description can be empty

File: src/report/createTicketForm/createTicketForm.vue [hunk 3]
```diff
@@ -118,20 +119,26 @@ export default class FindingTicketCreationForm extends Vue {
...
+      }
+    }
+    if (finding.score) {
+      description += `* Severity: ${severityText(finding.score)}\n`;
+    }
+    if (finding.affectedResource) {
+      description += `* Affected Resource: ${finding.affectedResource}\n`;
+    }
+    if (finding.details) {
+      description += `* Details: \n{noformat}${finding.details}\n{noformat}\n`;
```
Comment (jesusfcr) [inline src/report/createTicketForm/createTicketForm.vue:137]: I would add \n after first noformat

**Review by jesusfcr — CHANGES_REQUESTED** [review]

File: src/report/createTicketForm/createTicketForm.vue (hunk 2 above)
Comment (seilagamo) [reply src/report/createTicketForm/createTicketForm.vue]: Fixed in 857e089297916f825137faef1c3f533fd3059fb1

File: src/report/createTicketForm/createTicketForm.vue (hunk 3 above)
Comment (seilagamo) [reply src/report/createTicketForm/createTicketForm.vue]: Fixed in 857e089297916f825137faef1c3f533fd3059fb1

File: src/report/createTicketForm/createTicketForm.vue (hunk 1 above)
Comment (seilagamo) [reply src/report/createTicketForm/createTicketForm.vue]: Fixed in 857e089297916f825137faef1c3f533fd3059fb1

**Review by jesusfcr — APPROVED** [review]
LGTM

**Pull request merged**